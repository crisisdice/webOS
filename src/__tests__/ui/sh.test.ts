import { runAll } from '../ui'
import { sh } from '../cases'

runAll(sh)
