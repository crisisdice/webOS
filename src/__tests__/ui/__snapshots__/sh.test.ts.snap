// Vitest Snapshot v1

exports[`<LEFT 3><RIGHT 3> 1`] = `
"<body><div><div><div><div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`echo<ENTER>ls<ENTER>ArrowUpArrowUpArrowDown 1`] = `
"<body><div><div><div><div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ echo</span></div> <div></div><div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ ls</span></div> <div>.bashrc .history</div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span>ls<span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`g<LEFT 1> 1`] = `
"<body><div><div><div><div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span><span class=\\"caret-full svelte-136ugrd\\">g</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`hello<LEFT 1><RIGHT 2> 1`] = `
"<body><div><div><div><div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span>hello<span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`hello<LEFT 2><RIGHT 1> 1`] = `
"<body><div><div><div><div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span>hell<span class=\\"caret-full svelte-136ugrd\\">o</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`hello<LEFT 3><RIGHT 3> 1`] = `
"<body><div><div><div><div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span>hello<span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`hello<LEFT 5><RIGHT 6> 1`] = `
"<body><div><div><div><div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span>hello<span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`hello<LEFT 5><RIGHT 7> 1`] = `
"<body><div><div><div><div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span>hello<span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`hello<LEFT 6><RIGHT 4> 1`] = `
"<body><div><div><div><div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span>hell<span class=\\"caret-full svelte-136ugrd\\">o</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;

exports[`hello<LEFT 7><RIGHT 7> 1`] = `
"<body><div><div><div><div></div> <div class=\\"prompt svelte-1ykk7yx\\"><span><span class=\\"usr svelte-1ykk7yx\\">guest@127.0.0.1</span>:<span class=\\"pwd svelte-1ykk7yx\\">~/</span>
    $ <span>hello<span class=\\"caret-empty svelte-136ugrd\\">_</span></span></span></div></div> <input autofocus=\\"\\"></div></div></body>"
`;
